src/OrchardCore.Themes/TheAgencyTheme/TheAgencyTheme.csproj

```diff
@@ -11,18 +11,22 @@
     <None Remove="wwwroot\node_modules\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Include="Assets\scss\bootstrap-oc.scss" />
+    <EmbeddedResource Include="Assets\scss\modules\_flow.scss" />
+    <EmbeddedResource Include="Assets\scss\modules\_messages.scss" />
+    <EmbeddedResource Include="Assets\scss\modules\_pager.scss" />
+    <EmbeddedResource Include="Assets\scss\modules\_widgets.scss" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Theme.Targets\OrchardCore.Theme.Targets.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.DisplayManagement\OrchardCore.DisplayManagement.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement.Abstractions\OrchardCore.ResourceManagement.Abstractions.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Remove="**\*.less;**\*.scss;" />
-    <EmbeddedResource Remove="wwwroot\gulpfile.js" />
-    <EmbeddedResource Remove="wwwroot\index.html" />
-    <EmbeddedResource Remove="wwwroot\package-lock.json" />
-    <EmbeddedResource Remove="wwwroot\package.json" />
+    <Folder Include="wwwroot\scss\" />
```

Comment:
Needed?

Reply:
Ok, removed.

Reply:
@agriffard 

We exclude from embedded resources all files under the `Assets` folder **but nothing is excluded under `wwwroot`**, unless some well known things as `node_modules` that we exclude from everywhere.

So if a theme has its own asset building logic under its wwwroot folder, there can be more or less files that need to be explicitly excluded, as here `package-lock.json`. 

So i think we need to keep these specific exclusions

You can try it and check the size of the generated assembly. Good to test it with all the possible files under wwwroot, e.g after having installed all the needed things to build the asset pipeline.

Reply:
Those files are not here anymore in wwwroot.
The package.json is now at the root of the theme and restores the files in node_modules.

Reply:
Ah okay but we also embed files that are at the root of the module / theme, unless well known things as `node_module` or e.g csharp files, only files under the `Assets` folder are not embedded.

So we have to also explicitly exclude root files if they are quite big as e.g `package-lock.json` if it is now at the root but still there.

Reply:
@agriffard so here would be (maybe idem for other themes / modules)

    <EmbeddedResource Remove="package-lock.json" />

That said currently the size of this file is 267 ko while the size of the agency theme assembly (debug one so bigger) is 9680 ko

Reply:
Well, I did it but package-lock.json is just 1k because it just needs bootstrap and startbootstrap-agency, while the previous package.json in wwwroot needed all the gulp dependencies.

Reply:
Ah okay, indeed not needed for low size files